src/promptflow/promptflow/_cli/_pf/_run.py

```diff
@@ -91,6 +94,10 @@ def add_run_create_common(subparsers, add_param_list, epilog: Optional[str] = No
     )
     add_param_name = lambda parser: parser.add_argument("-n", "--name", type=str, help="Name of the run.")  # noqa: E731
 
+    add_param_resume_from = lambda parser: parser.add_argument(  # noqa: E731
```

Comment:
Update changelog1.7.0 after release (also the doc)

Reply:
lgtm after you update the changelog :D, pfs resume support in next patch?

Reply:
yes, will be a separate pr